tests/test_dtmf_aggregator.py

```diff
@@ -69,6 +70,7 @@ async def test_timeout_aggregation(self):
             aggregator,
             frames_to_send=frames_to_send,
             expected_down_frames=expected_down_frames,
+            pipeline_params=PipelineParams(allow_interruptions=False),
```

Comment:
This is necessary because `DTMFAggregator` pushes a `BotInterruptionFrame`, so we just need to ignore the `StartInterruptionFrame` now that it's pushed from the task.